### ISSUE TITLE
Install error for cffi on pypy3 (py 3.6.1)

### ISSUE THREAD

**Issue by andy-maier** [zhmcclient/zhmc-ansible-modules#130]
On pypy3 (currently Python 3.6.1) with minimum package levels, installation of cffi during 'make develop' failed with:

```
Could not find a version that satisfies the requirement cffi!=1.11.3,>=1.8 (from versions: )
No matching distribution found for cffi!=1.11.3,>=1.8
```

Travis log: https://travis-ci.org/github/zhmcclient/zhmc-ansible-modules/jobs/683180866

```
$ python --version
Python 3.6.1 (784b254d6699, Apr 14 2019, 10:22:42)
[PyPy 7.1.1-beta0 with GCC 6.2.0 20160901]

$ pip --version
pip 19.0.3 from /home/travis/virtualenv/pypy3.6-7.1.1/site-packages/pip (python 3.6)
```

The cffi package was installed with version 1.12.3 when this happened, so this points to an issue with parsing the version constraint "cffi!=1.11.3,>=1.8".

**Comment by andy-maier** [comment]
Trying different versions of pip shows that pip 18.0 and higher solves the problem.

**Comment by andy-maier** [comment]
Fixed in PR #129
